src/hooks/useSubaccount.tsx

```diff
@@ -877,6 +878,43 @@ const useSubaccountContext = ({ localDydxWallet }: { localDydxWallet?: LocalWall
     [localDydxWallet, compositeClient]
   );
 
+  const getVaultAccountInfo = useCallback(async () => {
+    const result = await compositeClient?.validatorClient.get.getMegavaultOwnerShares(dydxAddress);
+    if (result == null) {
+      return result;
+    }
+    return new DydxChainTransactions().parseToPrimitives(result);
+  }, [compositeClient?.validatorClient.get, dydxAddress]);
+
+  const depositToMegavault = useCallback(
+    async (amount: number) => {
+      if (subaccountClient == null) {
+        return undefined;
+      }
```

Comment:
instead of return undefined, do we want to actually `throw` an error so that it can be logged from `try/catch`?